src/loaders/compute_tools/gtdb_tk/gtdb_tk.py

```diff
@@ -33,6 +37,38 @@ def _run_gtdb_tk(ids_to_files: Dict[Path, str], output_dir: Path, threads: int,
     print(
         f'Used {round((end_time - start) / 60, 2)} minutes to execute gtdbtk classify_wf for '
         f'{len(ids_to_files)} genomes')
+    
+    fatal_dict = {}
+    summary_file_exists = False
+    summary_files = [loader_common_names.GTDBTK_AR53_SUMMARY_FILE,
+                     loader_common_names.GTDBTK_BAC120_SUMMARY_FILE]
+    selected_cols = [loader_common_names.GENOME_ID_COL,
+                     loader_common_names.CLASSIFICATION_COL]
+    for summary_file in summary_files:
+        summary_file_path = os.path.join(output_dir, summary_file)
+        if not os.path.exists(summary_file_path):
+            continue
+        try:
+            summary_df = pd.read_csv(summary_file_path, sep='\t', usecols=selected_cols)
+        except Exception as e:
+            raise ValueError(f"{summary_file} exists, but unable to retrive") from e
+        summary_file_exists = True
+        for genome_id, classfiy_res in zip(summary_df[loader_common_names.GENOME_ID_COL],
```

Comment:
So the GENOME_ID_COL is used globally in this code, but here it's being used to process a file that's not produced by our code. That makes me very nervous that this is just a coincidence. If it is, we shouldn't use this global name for this case.

Reply:
That file is written by gtdbtk, right? And it's just a coincidence that we use that column name in our own files?

Reply:
If so, then we should separate the usage, whether that means a new constant like GTDB_GENOME_ID_COL or something else

Reply:
Maybe because of naming issue, but it is a bunch of new constants I created specific for summary.tsv file.

Reply:
and summary.tsv is written by GTDB tk?

Reply:
OK, I took another look and I misread how the constant was used in one place. That being said, for any constants that are specific to the GTDB summary file, let's prefix them with `GTDB_` like you do for the files.

Reply:
Yes, summary.tsv is written/produced by GTDB tk tool. Sounds good to me! Will add GTDB_ prefix to relevant constants!!

Reply:
Updated!

Reply:
: